cpp/ps0004/ps0004_sol.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+int main()
+{
+    // prompting user for number 1/1.5
+    float number;
+    std::cout<<"Enter the number from 32 to 126\n";
+    std::cin>>number;
+    int temp_number = (int)number;
+    char opposite_characer = temp_number;
```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input